cardano-node/src/Cardano/Node/Protocol/Cardano.hs

```diff
@@ -182,16 +187,7 @@ mkSomeConsensusProtocolCardano NodeByronProtocolConfiguration {
           -- version that this node will declare that it understands, when it
           -- is in the Mary era. Since Mary is currently the last known
```

Comment:
The last sentence of this comment is now out of date. It should be:

> That is, it is the version of protocol /after/ Mary, i.e. Alonzo.

Reply:
Good catch :+1: